### ISSUE TITLE
Spanish account chart template not compatible with v10

### ISSUE THREAD

**Issue by anebi** [frappe/erpnext#12344]
Spanish account chart template not compatible with v10. When i copy es_account_chart_template_common.json to verified dir and try to install erpnext, I get error in last step. If i remove the file and try again with installation process, then setup complete fine. It looks like es_account_chart_template_common.json is not updated and is not compatible with v10.

May I ask you to have a look at that file and update it so I can use it on our new installations?

Thanks and regards,
Ali Nebi

**Comment by tundebabzy** [comment]
Hi @anebi 
Let's move this to the forum (https://discuss.erpnext.com). Please don't forget to add the specific error stack trace

**Comment by anebi** [comment]
Hi,

Thank you for your reply. I opened a thread in forum here (https://discuss.erpnext.com/t/spanish-account-chart-template-not-compatible-with-v10/32558), but for now no reply.

May I ask you to have a look? I did several tests and I am still not able to solve the problems completely. I put some comments in the thread after my tests, what I have done and achieved so far, but I am stuck in this process.

Thanks and regards,
Ali Nebi
